tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/internal/findandreplace/WidgetExtractor.java

```diff
@@ -0,0 +1,90 @@
+package org.eclipse.ui.internal.findandreplace;
```

Comment:
Missling licenseheader. Please make sure it's there before pushing in.

Reply:
Thanks for the hint!